omero/sysadmins/version-requirements.txt

```diff
@@ -87,37 +69,32 @@ Bitness
 Currently 32- and 64-bit systems are supported for client and server
 on all platforms with the exception of MacOS X (64-bit only).
 
-* [5.1] Drop 32-bit support for server, retain 32-bit support for
-  clients
-* [5.2] Drop 32-bit support for Ice and native code
+* [5.2] Drop 32-bit support for Ice and native code [server]
+* [5.3] Deprecate 32-bit support for Ice and native code [client]
 * Rationale: It’s not practical to run a server on a 32-bit system due
   to its memory requirements, so it’s safe to drop 32-bit support for
-  OMERO.server in 5.1 given that it’s de-facto 64-bit-only
+  OMERO.server in 5.2 given that it’s de-facto 64-bit-only
   today. There are still a significant number of 32-bit Windows
   clients (~60%), so support should be retained until this changes.
   Maintaining custom builds of Ice for 32-bit systems is extremely
-  costly, and so we could consider dropping them for 5.2.
+  costly, and so we could consider dropping them for 5.3.  This also
+  depends upon Ice support for 32-bit systems.
 
 Components
 ----------
 
 * PostgreSQL
 
-  * [5.1] 8.4/9.0/9.1 dropped, 9.2 deprecated, 9.3+ recommended
   * [5.2] 9.2 dropped, 9.3 deprecated, 9.4+ recommended
+  * [5.3] 9.2 dropped, 9.3 deprecated, 9.4+ recommended
   * Rationale: Current releases available for all supported systems
     from upstream and for CentOS/RHEL 6.x officially via SCL (software
-    collections). At a bare minimum, 8.4 and 9.0 must be dropped (end
-    of security support); dropping 9.1 removes the floating point
-    issues fixed by 9.2+. Due to its age, 8.4 no longer receives
-    adequate testing and so we can’t claim to support it. The same
-    applies to 9.0 and 9.1.  Given the annual release cycle, it would
-    make sense to deprecate 9.2 for 5.2.
+    collections).
 
 * Python
 
-  * [5.1] 2.6 deprecated, 2.7 recommended, 3.x not supported
   * [5.2] 2.6 dropped, 2.7 recommended, 3.x not supported
+  * [5.3] 2.6 dropped, 2.7 recommended, 3.x not supported
   * Rationale: 2.7 is provided by all systems except for CentOS/RHEL
     6.x, however it is available officially via SCL so could be made
```

Comment:
It is speculative to some extent.  Dropped also doesn't mean "won't work" but "unsupported and use at own risk", which is exactly where django 1.6 is at, and so reflects the same risk.